rest-api/src/controllers/AuthController.ts

```diff
@@ -42,6 +56,16 @@ export class AuthController implements IController {
             ...this.signInValidation,
             this.signIn,
         );
+        router.post(
+            `${basePath}/sign-up`,
+            ...this.signUpValidation,
+            this.signUp,
+        );
+        router.get(
```

Comment:
There is no signIn yet? You just did the signin with the grpc and then tested this with that token?

Reply:
There already was sign in in place 😄